### PR TITLE
Integration with krkn-hub

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description

krkn-hub integration setup:

1. 'krknhub_workload' e.g. node-scenarios
For more scenarios, [see the documentation](https://github.com/krkn-chaos/krkn-hub?tab=readme-ov-file#supported-chaos-scenarios.)

2. 'krknhub_environment_variable' e.g. 
export CLOUD_TYPE= 
export BMC_USER=
export BMC_PASSWORD=
export BMC_ADDR=
export NODE_NAME=
export ENABLE_ES=
export ES_SERVER=
export ES_PORT=
export ES_METRICS_INDEX=
export ES_COLLECT_METRICS=

3. Save krkn-hub pod logs

### CODE REVIEW

**Review by RobertKrawitz — COMMENTED** [review]
I don't see krknhub_exceptions.py used anywhere?

Otherwise, I'm OK with this.

**Comment by ebattat** [comment]
Yes, currently I didnt use it but its a place holder for the future

**Comment by RobertKrawitz** [comment]
> Yes, currently I didnt use it but its a place holder for the future

Could you add a comment to it to that effect, then?

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
krkn_exceptions.py needs a comment that the file is currently unused but is added as a placeholder for the future.

**Comment by ebattat** [comment]
You right, added a Base class for all KrknHub error classes

**Review by RobertKrawitz — APPROVED** [review]

File: benchmark_runner/main/environment_variables.py [hunk 1]
```diff
@@ -190,6 +191,11 @@ def __init__(self):
         self._environment_variables_dict['clusterbuster_workload'] = EnvironmentVariables.get_env('CLUSTERBUSTER_WORKLOAD', '')
         self._environment_variables_dict['clusterbuster_uuid'] = EnvironmentVariables.get_env('CLUSTERBUSTER_UUID', '')
 
+        # KRKN HUB data: Chaos testing
+        # For more details, see the documentation: https://github.com/krkn-chaos/krkn-hub?tab=readme-ov-file#supported-chaos-scenarios.
+        self._environment_variables_dict['krknhub_workload'] = EnvironmentVariables.get_env('KRKNHUB_WORKLOAD', '')
+        self._environment_variables_dict['krknhub_environment_variables'] = EnvironmentVariables.get_env('KRKNHUB_ENVIRONMENT_VARIABLES', '')
```
Comment (RobertKrawitz) [inline benchmark_runner/main/environment_variables.py:197]: What is this variable for and what does the value look like?

Comment (ebattat) [reply benchmark_runner/main/environment_variables.py:197]: See the krknhub_environment_variables in Description

Comment (RobertKrawitz) [reply benchmark_runner/main/environment_variables.py:197]: I don't see mention in the PR description (and there should be a description in the code, not just in the PR).

Comment (ebattat) [reply benchmark_runner/main/environment_variables.py:197]: See it up:
'krknhub_environment_variable' e.g.
export CLOUD_TYPE=
export BMC_USER=
export BMC_PASSWORD=
export BMC_ADDR=
export NODE_NAME=
export ENABLE_ES=
export ES_SERVER=
export ES_PORT=
export ES_METRICS_INDEX=
export ES_COLLECT_METRICS=

The environment variable is changes between the test scenarios,
I have added the document link:
        # For more details, see the documentation: https://github.com/krkn-chaos/krkn-hub?tab=readme-ov-file#supported-chaos-scenarios.

Comment (RobertKrawitz) [reply benchmark_runner/main/environment_variables.py:197]: Right, but what does the exact value of KRKNHUB_ENVIRONMENT_VARIABLES look like?  Environment variables are single strings, not a list of strings; if you want a list, it has to be encoded (typically colon separated, in the case of PATH and similar).

Comment (ebattat) [reply benchmark_runner/main/environment_variables.py:198]: Good point: Added
`# e.g. "export CLOUD_TYPE='test'; export BMC_USER='user'"`

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**